src/app/layout.js

```diff
@@ -1,10 +1,20 @@
-import "@/app/global.css";
+import '@/app/global.css';
+import { Roboto } from 'next/font/google';
+
+const roboto = Roboto({ weight: ['400', '700'], subsets: ['latin'] });
```

Comment:
```suggestion
const roboto = Roboto({ weight: ['400', '700'], subsets: ['latin'], preload: false, });
```
Fixes a warning about prefetching a font file.

Reply:
`The resource at “http://localhost:3000/_next/static/media/ce62453a442c7f35-s.p.a9507876.woff2” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly.`